Demo/Demo/Models/VaultSessionID.swift

```diff
@@ -0,0 +1,13 @@
+struct VaultSessionID: Codable {
+
+    let status: String?
+    let links: [LinkResponse]
+}
+
+struct LinkResponse: Codable {
+
+    let href: String?
+    let rel: String?
+    let method: String?
+    let encType: String?
```

Comment:
Can we expand the abbreviation for `encType` to better fit the [Swift Naming Guidelines](https://www.swift.org/documentation/api-design-guidelines/#avoid-abbreviations)?

Reply:
`encType` comes directly from the Orders API, i dont even know what `encType` is :P

Reply:
Does our demo need this field at all? I'm pretty sure if we just remove the fields we don't need from the JSON response, `Codable` is fine to ignore them if omitted from this struct, while still parsing the rest.